### DISCUSSION TITLE
Hey, I want help for google cloud map style.

### DISCUSSION THREAD

**Discussion by bhomaramgtsinfo** [react-native-maps/react-native-maps#4748] [Q&A]
Hi, I have done some research and found that we can create custom style on google cloud console. But if we want to use that custom style in our map then we have to use mapId props but i am not able to use mapId in our react native map. Please help me how can i use Map ID in our react native maps.

**Comment by localjo** [comment]
I would also like to be able to use a `mapId` but it seems that it isn't supported by this library yet. I'm unfortunately still needing to use the legacy JSON wizard and define a `customMapStyle`. It seems that Google is trying to phase that out though, so eventually this library will probably need to add `mapId` support.
